modules/manage/pages/manage-nodes/apply-node-to-node-encryption.adoc

```diff
@@ -74,7 +74,10 @@ The output indicates that encryption has been successfully enabled for each node
 
 . Establish an appropriate encryption-level for the cluster.
 This is achieved by the `setting-security` CLI command, specifying the `--cluster-encryption-level` parameter.
-Its value can be either control (meaning that only server-management information is passed in encrypted form) or all (meaning that all information, including data handled by services, is passed in encrypted form):
+Its value can be `control`, meaning that server-management information passed between nodes is passed in encrypted form; `all`, meaning that all information passed between nodes, including data handled by services, is passed in encrypted form; or `strict`, meaning the same as `all`, but meaning additionally that only encrypted communication is permitted between nodes and between the cluster and external clients.
```

Comment:
Same comment as above.

Reply:
I've changed for all to:
`strict`, meaning `all` with only encrypted communication permitted between nodes and between the cluster and external clients.